server/config/config.go

```diff
@@ -83,20 +85,32 @@ func (conf Config) StaticURL(appID apps.AppID, name string) string {
 	return conf.AppURL(appID) + "/" + path.Join(appspath.StaticFolder, name)
 }
 
+// developerModeDomains is the list of domains for which DevelopmentMode will be
+// forced on. Empty for the time being.
+var developerModeDomains = regexp.MustCompile("^" + strings.Join([]string{}, "|") + "$")
```

Comment:
1/5 that DevelopmentMode should never be enabled by in an instance on the public internet. What use case did you had in mind?

Reply:
My dev instance is on ngrok, therefore on the public internet. Having said that, I do not have a specific use case, leftover code that I was hesitant to remove, no problem removing altogether.